dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,690 @@
+the same project, and the *builder* service account has rights to push any image
+in the same project.
+
+[[using-image-pull-secrets]]
+== Using Image Pull Secrets
+
+link:../architecture/infrastructure_components/image_registry.html[Docker
+registries] can be secured to prevent unauthorized parties from accessing
+certain images. If you are link:#accessing-the-internal-registry[using
+{product-title}'s internal registry] and are pulling from image streams located
+in the same project, then your pod's service account should already have
+permissions and no additional action should be required. If this is not the
+case, then additional configuration steps are required.
+
+[[allowing-pods-to-reference-images-across-projects]]
+=== Allowing Pods to Reference Images Across Projects
+
+When using the internal registry, to allow pods in *project-a* to reference
+images in *project-b*, a service account in *project-a* must be bound to the
+`system:image-puller` role in *project-b*:
+
+----
+$ oc policy add-role-to-user \
+    system:image-puller system:serviceaccount:project-a:default \
+    --namespace=project-b
+----
+
+After adding that role, the pods in *project-a* that reference the default
+service account will be able to pull images from *project-b*.
+
+To allow access for any service account in *project-a*, use the group:
+
+----
+$ oc policy add-role-to-group \
+    system:image-puller system:serviceaccounts:project-a \
+    --namespace=project-b
+----
+
+[[allowing-pods-to-reference-images-from-other-secured-registries]]
+=== Allowing Pods to Reference Images from Other Secured Registries
+
+To pull a secured Docker image that is not from OpenShift's internal registry,
+you must create a *dockercfg* secret and add it to your service account.
+
+If you already have a *_.dockercfg_* file for the secured registry, you can
+create a secret from that file by running:
+
+----
+$ oc secrets new <pull_secret_name> .dockercfg=<path/to/.dockercfg>
+----
+
+If you do not already have a *_.dockercfg_* file for the secured registry, you
+can create a secret by running:
+
+----
+$ oc secrets new-dockercfg <pull_secret_name> \
+    --docker-server=<registry_server> --docker-username=<user_name> \
+    --docker-password=<password> --docker-email=<email>
+----
+
+To use a secret for pulling images for pods, you must add the secret to your
+service account. The name of the service account in this example should match
+the name of the service account the pod will use; *default* is the default
+service account:
+
+----
+$ oc secrets add serviceaccount/default secrets/<pull_secret_name> --for=pull
+----
+
+To use a secret for pushing and pulling build images, the secret must be
+mountable inside of a pod. You can do this by running:
+
+----
+$ oc secrets add serviceaccount/builder secrets/<pull_secret_name>
+----
+
+[[creating-an-image-stream-by-manually-pushing-an-image]]
+== Creating an Image Stream by Manually Pushing an Image
```

Comment:
@soltysh @miminar On 2nd thought, maybe this should actually be the final subsection under the "Importing Tag and Image Metadata" section?